packages/@yoda/util/fs.js

```diff
@@ -34,3 +35,23 @@ function mkdirp (dir, callback) {
     }) /** fs.stat */
   }) /** fs.mkdir */
 }
+
+/**
+ * @param {string} dir
+ */
+function mkdirpSync (dir) {
+  try {
+    fs.mkdirSync(dir)
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      return mkdirpSync(path.dirname(dir))
```

Comment:
Suppose a call `mkdirSync ('/aaa/bbb/ccc/ddd')`, and all dirs are not exist, after call, the result is only make a root dir '/aaa' ?

Reply:
Nope, the behavior of `mkdir` is consistent with Linux's mkdir, so the call won't make any directory, instead of throwing an `Error`, too.

Reply:
> Nope, the behavior of `mkdir` is consistent with Linux's mkdir, so the call won't make any directory, instead of throwing an `Error`, too.

I mean the code may has problem. The intent is to recursive create all parent directories but the result will only create a root one if all directories are not exists.